core/api_requests.py

```diff
@@ -50,36 +51,15 @@ def get_history(id):
     response = requests.post(URL, json={'query': query_history(id)})
 
     return response.json()
-
 def get_rockets():
     """
     request rockets information
     """
     response = requests.post(URL, json={'query': query_rockets()})
     return response.json()
-
 def get_rocket(id):
     """
     request single rocket information
     """
     response = requests.post(URL, json={'query': query_rocket(id)})
-    return response.json()
```

Comment:
## pt

Aqui vc está removendo o retorno da função, isso quebrará o código

## en

Here you are removing the function return data, this will break the code